packages/server-wallet/src/utilities/workers/loader.js

```diff
@@ -1,4 +1,4 @@
 const path = require('path');
 
-require('ts-node').register();
+require('ts-node').register({typeCheck: false});
```

Comment:
For speed here we may want to use something like this https://github.com/alangpierce/sucrase

Reply:
FYI this was introduced not for speed, but because `ts-node` was complaining about a type error that it doesn't flag when compiling normally. (Possibly it is not using the project's tsconfig?)


I am reluctant to have the published library require any kind of transpilation at runtime in any case. It seems like it should be all JavaScript.